<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dif</class>
 <widget class="QWidget" name="dif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>误差展示</string>
   </property>
  </widget>
  <widget class="QChartView" name="my_chart" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="my_chart_2" native="true">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="my_chart_3" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
